### ISSUE TITLE
Build asset is "dirty"

### ISSUE THREAD

**Issue by dklimpel** [Awesome-Technologies/synapse-admin#347]
I do not know if is it intentional or a mistake that the build asset has the name: `synapse-admin-0.8.7-dirty.tar.gz`

Reason is the build job:

https://github.com/Awesome-Technologies/synapse-admin/blob/9b43d41040b79cc8ad240dbadc66098bc8231c2b/.github/workflows/github-release.yml#L23

The paramater `--dirty` adds `-dirty` to the name.

**Comment by meramsey** [comment]
agreed it also makes the folder inside the same way which breaks automated stuff which pulls tar.gz down as internal folder needs renamed too

**Comment by damluk** [comment]
`man git describe`:

> If the working tree has local modification "-dirty" is appended to it

And indeed, `yarn install` and `yarn build` modify `yarn.lock`:

[dirty.log](https://github.com/Awesome-Technologies/synapse-admin/files/13538651/dirty.log)

I cannot reproduce it locally though, neither in a `node:16` nor in a `ubuntu:22.04` container with matching `yarn`, `npm` and `node` versions.

**Comment by awesome-manuel** [comment]
Need to use `yarn install --immutable`.
